botnim/vector_store/vector_store_es.py

```diff
@@ -6,8 +6,8 @@
 
 from elasticsearch import Elasticsearch
 from openai import OpenAI
-from ..config import get_logger
-from ..config import DEFAULT_EMBEDDING_MODEL, DEFAULT_EMBEDDING_SIZE
+from botnim.config import get_logger
```

Comment:
I think relative imports work better in the context of a single library, why the change?